packages/api-base/src/types/submittable.ts

```diff
@@ -68,13 +68,17 @@ export interface SubmittableExtrinsic<ApiType extends ApiTypes, R extends ISubmi
   send (statusCb: Callback<R>): SubmittableResultSubscription<ApiType, R>;
 
   /**
-   * @description Sign and broadcast the constructued transaction.
+   * @description Sign the constructed transaction asynchronously.
+   *
+   * The result is a signed extrinsic that is ready to be broadcast to the network via `.send()`, `rpc.author.submitExtrinsic()`, or
+   * any custom submission logic.
    *
    * Note for injected signers:
-   * As of v12.0.2 and up the `SignerResult` return type for `signPayload` allows for the `signedTransaction` field.
-   * This allows the signer to input a signed transaction that will modify the payload. This
-   * The api will ensure that the Call Data is not changed. This allows for the signer to modify the payload to add
-   * things like `mode`, and `metadataHash` for signedExtensions such as `CheckMetadataHash`.
+   * As of v12.0.1 and up the `SignerResult` return type for `signPayload` allows for the `signedTransaction` field.
+   * This allows the signer to input a signed transaction that will be directly broadcasted. This
```

Comment:
>    * This allows the signer to input a signed transaction that will be directly broadcasted. This
   * bypasses the api adding the signature to the payload. The api will ensure that the Call Data is not changed before it broadcasts the
   * transaction.
   
   This still needs to be adjusted right?